javascript-local-storage/data.js

```diff
@@ -0,0 +1,11 @@
+/* exported todos */
+
+var todos = [];
+// eslint-disable-next-line no-unused-vars
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = (JSON.parse(localStorage.getItem('javascript-local-storage')));
```

Comment:
You should check if there is anything stored in `previousTodosJSON` before reassigning the value of `todos`. If there isn't anything there, the value will be `null`, which means you would be destroying the array.